### PR TITLE
Implemented creation of model deployments endpoints

### PR DESCRIPTION
**Pull request by HermanPlay**
Added to endpoints to model router.

### CODE REVIEW

File: app/api/routers/model.py [hunk 1]
```diff
@@ -77,6 +78,71 @@ async def put_model(model_data: model_schemas.PutModel, db: Session = Depends(ge
...
+        raise HTTPException(status_code=404, detail="Model not found!")
+
+    if db_model.status == Status.ACTIVE:
+        raise HTTPException(status_code=409, detail="Model already active!")
+
+    db_model_details = ModelDetailsService.get_model_details_by_model_id(db, model_id)
+    if not db_model_details:
+        raise HTTPException(status_code=404, detail="Model details not found!")
+
+    for key, val in db_model_details.__dict__.items():
```
Comment (matined) [inline app/api/routers/model.py:110]: use `any()` function instead of for loop

File: app/api/utils/model_deployment.py [hunk 2]
```diff
@@ -64,12 +54,12 @@ def __deploy_to_k8s(self) -> None:
         v1.create_namespaced_deployment(
             namespace=Constants.K8S_NAMESPACE_MODELS, body=deployment
         )
-        print("Deployment ready")
+        _logger.info("Deployment ready")
```
Comment (matined) [inline app/api/utils/model_deployment.py:57]: if we want to log it, please add at least deployment's name to the log

File: app/api/services/model.py [hunk 3]
```diff
@@ -108,3 +131,63 @@ def delete_model(cls, db: Session, model_id: int) -> JSONResponse:
...
+
+        :param db: Database session
+        :param model_id: id of model to activate
+
+        :return: JSON respose indicating succesful activation
+        """
+
+        model_deployment = ModelDeployment(
+            name=name,
+            model_details={**model_details},
```
[the remark this answers is not in the document]
Comment (matined) [reply app/api/services/model.py:152]: @HermanPlay what about this one?

[the remark this answers is not in the document]
Comment (HermanPlay) [reply app/api/services/model.py]: I changed it, no we directly pass ModelDetails object

**Review by matined — APPROVED** [review]

**Pull request merged**